phpBB/includes/functions_upload.php

```diff
@@ -64,6 +69,8 @@ class filespec
 	*/
 	function filespec($upload_ary, $upload_namespace, \phpbb\mimetype\guesser $mimetype_guesser = null, \phpbb\plupload\plupload $plupload = null)
 	{
+		global $phpbb_filesystem;
```

Comment:
No idea, but that's in includes, so i guess global is fine, if I put it there i have no idea why.